hack/verify-openapi-spec.sh

```diff
@@ -32,10 +32,10 @@ _tmp="${KUBE_ROOT}/_tmp"
 
 mkdir -p "${_tmp}"
 cp -a "${SPECROOT}" "${TMP_SPECROOT}"
-trap "cp -a ${TMP_SPECROOT} ${SPECROOT}/..; rm -rf ${_tmp}" EXIT SIGINT
-rm ${SPECROOT}/*
-cp ${TMP_SPECROOT}/BUILD ${SPECROOT}/BUILD
-cp ${TMP_SPECROOT}/README.md ${SPECROOT}/README.md
+trap 'cp -a "${TMP_SPECROOT}" "${SPECROOT}/.."; rm -rf "${_tmp}"' EXIT SIGINT
```

Comment:
change reason:
```
In hack/verify-openapi-spec.sh line 35:
trap "cp -a '${TMP_SPECROOT}' '${SPECROOT}'/..; rm -rf '${_tmp}'" EXIT SIGINT
             ^-- SC2064: Use single quotes, otherwise this expands now rather than when signalled.
                               ^-- SC2064: Use single quotes, otherwise this expands now rather than when signalled.
                                                        ^-- SC2064: Use single quotes, otherwise this expands now rather than when signalled.
```